shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeSyncService.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.shard.operator.v2.converters.ManagedBridgeConverter;
+import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+
+@ApplicationScoped
+public class ManagedBridgeSyncService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ManagedBridgeSyncService.class);
+
+    @Inject
+    ManagerClient managerClient;
+
+    @Inject
+    ManagedBridgeService managedBridgeService;
+
+    @Inject
+    NamespaceProvider namespaceProvider;
+
+    public void syncManagedBridgeWithManager(){
+        managerClient.fetchBridgesToDeployOrDelete()
+                .onItem()
+                .invoke(this::processDelta)
+                .subscribe().with(
+                        success -> LOGGER.debug("Successfully process ManagedBridge delta"),
```

Comment:
@vaibhavjainwiz Couple of minor typos here. I think they should be:

- `Successfully processed ManagedBridge deltas`
- `Failed to process ManagedBridge deltas`

We also seem to be losing the failure reason. Is it possible to print the Exception stacktrace when things fail?

Reply:
done